pkg/component/component.go

```diff
@@ -245,6 +245,12 @@ func CreateFromPath(client *occlient.Client, params occlient.CreateArgs) error {
 		return err
 	}
 
+	if params.Wait {
+		podSelector := fmt.Sprintf("deploymentconfig=%s-%s", labels["app.kubernetes.io/component-name"], labels["app"])
+		_, err = client.WaitAndGetPod(podSelector)
+		return err
```

Comment:
why aren't we checking the error and instead just returning it?

We have no error checking here... We should have:

```go
if err != nil {
		return err
}
s.End(true)
```